app/components/moderation-list/template.hbs

```diff
@@ -48,6 +62,22 @@
                                 }}
                             </div>
                         {{/if}}
+                        {{#if (and isRequests (eq requests.length 0))}}
+                            <div class="text-center p-v-md {{local-class 'moderation-list-row' from='reviews/components/moderation-list-row/styles'}}">
+                                {{t 'components.moderationList.noRequests'}}
+                            </div>
+                        {{else}}
+                            {{#each requests as |request|}}
+                                {{request-list-row request=request}}
+                            {{/each}}
+                            <div class="pull-right text-right">
+                                {{pagination-pager
+                                    count=totalPages
+                                    current=page
+                                    change=(action pageChanged)
+                                }}
+                            </div>
+                        {{/if}}
```

Comment:
Not sure the logic is correct here. It seems like submissions and requests will both always display. Please review.

Reply:
After looking at the code, here's what's happening. The `moderation-list` template is used in two routes: moderation (showing a list of preprint submissions) and withdrawal requests (showing a list of withdrawal requests). `isSubmissions` and `isRequests`  are used to distinguish between usage in these two different views. Therefore, `requests` will only be defined if `isRequests` is true, and `submissions` will be defined if `isSubmissions` is true.